src/types/tuple.rs

```diff
@@ -110,8 +112,8 @@ impl<'a> Iterator for PyTupleIterator<'a> {
 
     #[inline]
     fn next(&mut self) -> Option<&'a PyAny> {
-        if self.index < self.slice.len() {
-            let item = self.slice[self.index];
+        if self.index < self.tuple.len() {
```

Comment:
Just a nit, but I think it would be better to have `length` in `PyTupleIterator`.
`slice.len()` is zero-cost, but `tuple.len()` is not because of FFI boundary.

Reply:
Will do!